toolchain/sem_ir/file.h

```diff
@@ -301,6 +304,9 @@ class File : public Printable<File> {
   // List of Cpp imports.
   ValueStore<ImportCppId> import_cpps_;
 
+  // The Clang AST to used when looking up `Cpp` names.
+  clang::ASTUnit* cpp_ast_ = nullptr;
```

Comment:
Added this initialization since it's not initialized everywhere and no warning is given.
https://github.com/carbon-language/carbon-lang/blob/43b9969058b3de2e0b05896ef8e4489c1dc85cb6/toolchain/language_server/context.cpp#L146